packages/example-getting-started/docs/7-controller.md

```diff
@@ -1,71 +0,0 @@
-### Create your controller
```

Comment:
Are you intentionally removing `7-controller.md` file? Isn't this going to break Kevin's tutorial on loopback.io?

Reply:
@bajtos my bad rebase added it in the first commit, so I removed it in this fix commit, I checked https://github.com/strongloop/loopback-next/tree/master/packages/example-getting-started/docs, that doc should be renamed/replaced by `controller.md`, but I will double check with @kjdelisle